frontend/package-lock.json

```diff
@@ -2983,11 +2983,13 @@
             },
             "balanced-match": {
               "version": "1.0.0",
-              "bundled": true
```

Comment:
Unless we're changing `frontend/package.json`, let's ignore the automatic changes to `frontend/package-lock.json`. It can be risky to update your dependencies with every commit.